client/src/redux/reducers/relatedProductReducer.js

```diff
@@ -1,4 +1,45 @@
-const defaultState = [];
+const defaultState = [
```

Comment:
Is this just for development? If so, let's make sure we create a Trello card to remove it later on

Reply:
Yes! just added the card.